scheduler/pkg/agent/modelserver_controlplane/oip/v2.go

```diff
@@ -51,19 +50,12 @@ func CreateV2GrpcConnection(v2Config V2Config) (*grpc.ClientConn, error) {
 		grpc_retry.WithMax(v2Config.GRPRetryMaxCount),
 	}
 
-	kacp := keepalive.ClientParameters{
```

Comment:
could we add a note on why the keepalive was removed here?

Reply:
agent and model server are on the same pod/network and therefore no need to set keepalive